test/.themisrc

```diff
@@ -1,15 +1,19 @@
 let g:repo_root = fnamemodify(expand('<sfile>'), ':h:h')
 
 call themis#option('exclude', g:repo_root . '/test/README.md')
-call themis#option('exclude', g:repo_root . '/test/Guardfile')
```

Comment:
The `Guardfile` is a local file in my environment. It is not committed to repo, but exists in my local directory. So please do not remove this line.